backend/codesirius/problems/serializers/execution_constraint_v2.py

```
@@ -0,0 +1,203 @@
+
+
+class ExecutionConstraintListSerializer(serializers.ListSerializer):
+
+    @staticmethod
+    def snake_to_camel(snake_str):
+        components = snake_str.split("_")
+        return components[0] + "".join(x.title() for x in components[1:])
+
+    @staticmethod
+    def _validate_update(
+        attrs, constraint_id, existing_constraints_map
```

Ruff (0.8.2)

Missing type annotation for function argument `attrs`

(ANN001)